index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
Not sure if you care, but there's some wonky indentation in this file (link tag in header, div tags in old notes).